app/domain/authentication/authn_k8s/validate_pod_request.rb

```diff
@@ -38,7 +38,7 @@ def validate_webservice_exists
 
       def validate_host_can_access_service
         return if host_can_access_service?
-        raise SecurityErr::RoleNotAuthorizedOnWebservice.new(host.role.id, service_id)
+        raise SecurityErr::RoleNotAuthorizedOnWebservice.new(host.role.id, "authenticate", service_id)
```

Comment:
You might need to update specs too no? Ex: `validate_pod_request_spec` `validate_webservice_access_spec`

Reply:
No because the message didn't change.